src/common/datatypes/Edge.cpp

```diff
@@ -32,6 +32,45 @@ std::string Edge::toString() const {
   return os.str();
 }
 
+// format:
+// {
+//    "prop1": val1,
+//    "prop2": val2,
+// }
+folly::dynamic Edge::toJsonObj() const {
+  folly::dynamic propObj = folly::dynamic::object();
+
+  for (const auto& iter : props) {
+    propObj.insert(iter.first, iter.second.toJsonObj());
+  }
+
+  return propObj;
+}
+
+// Used in Json form query result
+// format:
+// {
+//   "id": {
+//      "name": _name,
+//      "src": srcVID,
+//      "dst": dstVID,
+//      "type": _type,
+//      "ranking": _rankding
+//    }
```

Comment:
"id" is used to be compatible with neo4j?

Reply:
Correct. In neo4j, `id` field for an edge is simply `"id": edgeID`.